firmware/config/stm32f4ems/efifeatures.h

```diff
@@ -257,10 +257,14 @@
 
 #define EFI_CONSOLE_USB_DEVICE SDU1
 
-// F42x has more memory, so we can:
-//  - use compressed USB MSD image (requires 32k of memory)
-//  - use perf trace (requires ~16k of memory)
-#ifdef EFI_IS_F42x
+#if defined(EFI_HAS_EXT_SDRAM)
+    #define ENABLE_PERF_TRACE TRUE
+    #define LUA_USER_HEAP (1 * 1024 * 1024)
```

Comment:
WOW that's so cool!

Reply:
Actually there is 8Mb. You can allocate whole SDRAM to LUA.

Reply:
nope, let's keep this as is with 1Mb for now :)